pkgs/development/python-modules/blis/default.nix

```diff
@@ -51,17 +59,14 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "blis" ];
 
-  passthru = {
-    # Do not update to BLIS 0.9.x until the following issue is resolved:
-    # https://github.com/explosion/thinc/issues/771#issuecomment-1255825935
```

Comment:
@davidak Can thinc's numpy pin be removed?

Reply:
Actually upstream removed it in https://github.com/explosion/thinc/commit/efda3b4e8f4bf7d8518d562feeb3321ae4920309 so this should be fine.